dietpi/dietpi-software

```diff
@@ -10720,11 +10723,16 @@ _EOF_
 [Unit]
 Description=DietPi Gogs service
 DefaultDependencies=no
+#Optional
+#After=mysqld.service
 
 [Service]
 Type=simple
 User=gogs
-ExecStart=/bin/bash -c '/etc/gogs/gogs web &> /var/log/gogs_daemon.log'
```

Comment:
Hmm, I don't remember any more why we used `/bin/bash -c 'command'` by default for our services 🤔. But I think your solution makes sense for everything that does not necessarily use `bash`. `dash` (`sh`) might be a tiny bid faster for scripts that that support it.